canmatrix/canmatrix.py

```diff
@@ -36,6 +36,13 @@
 import logging
 import fnmatch
 
+defaultFloatFactory = float
+try:
+    from decimal import *
+    defaultFloatFactory = Decimal
+except:
+    defaultFloatFactory = float
```

Comment:
A bit of redundancy here reassigning `defaultFloatFactory = float`.  Also, importing * is pretty hazardous as far as name collisions and hard to trace.  Maybe:

```python3
try:
    import decimal
except ImportError:
    defaultFloatFactory = float
else:
    defaultFloatFactory = decimal.Decimal
```

But...  what is the scenario where `decimal` is appropriately not available?  It seems it was added in Python 2.4.